public/views/login/LoginController.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+(function () {
+
+    var app = angular.module('CS4570');
+
+    app.controller('LoginController', ['$http', '$scope', '$window', '$filter', '$location', '$rootScope', '$cookies',
+        function ($http, $scope, $window, $filter, $location, $rootScope, $cookies) {
+            var self = this;
+
+            if($cookies.get('email')) {
+                $location.path("/home");
+            }
+
+
+            self.user = {
+                email: '',
+                password: ''
+            };
+
+
+            self.enableSubmit = function () {
+                return self.user.email != '' && self.user.password != '';
+            };
+
+            self.login = function () {
+                console.log(self.user);
+
+                $http.post($window._base_url + "sign_in", self.user)
+                    .then(function (result) {
+                        var res = result.data;
+                        console.log(res.data);
+                        console.log(res.token);
+
+                        $location.path('/home');
```

Comment:
You should set the cookies first before setting the location to home, otherwise it can cause a memory leak for still referencing the $cookies after instantiating another controller.